src/main/kotlin/org/axonframework/intellij/ide/plugin/inspections/saga/descriptions.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (c) 2022. Axon Framework
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.axonframework.intellij.ide.plugin.inspections.saga
+
+
+const val associationPropertyStaticDescription = """The message does not contain the specified associationProperty."""
+
+const val associationPropertyDescription =
+    """<html>The message does not contain the specified associationProperty. This is required by the @SagaEventHandler annotation.
```

Comment:
Suggestion to rephrase:

"The message does not declare a property for the specified associationProperty in the `@SagaEventHandler` annotation. Please ensure an accessor method is available for this property. Also see ..."

Reply:
Rephrased